src/test/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/HandTurnActiveTest.java

```diff
@@ -193,6 +193,59 @@ void in_the_second_match_the_start_player_shifts_by_1() {
     Assertions.assertThat(previousMatch.getHands()).allMatch(hand -> !hand.isTurnActive());
   }
 
+  @Test
+  void in_the_second_match_the_start_player_shifts_by_1_following_the_participation_number() {
+    Game game =
+        GameBuilder.builder("game1")
+            .withParticipation(PLAYER_1)
+            .withParticipation(PLAYER_2)
+            .withParticipation(PLAYER_3)
+            .withMatch()
+            .withMatchState(MatchState.FINISH)
+            .withRound()
+            .finishRound()
+            .finishMatch()
+            .withMatch()
+            .withMatchState(MatchState.PLAYING)
+            .withHandForPlayer(PLAYER_1)
+            .withCards(ACE_OF_CLUBS, QUEEN_OF_CLUBS)
+            .withAnnouncedScore(1)
+            .finishHand()
+            .withHandForPlayer(PLAYER_2)
+            .withCards(KING_OF_CLUBS, JACK_OF_CLUBS)
+            .withAnnouncedScore(0)
+            .finishHand()
+            .withHandForPlayer(PLAYER_3)
+            .withCards(QUEEN_OF_HEARTS, KING_OF_HEARTS)
+            .withAnnouncedScore(0)
+            .finishHand()
+            .withRound()
+            .finishRound()
+            .withRound()
+            .finishRound()
+            .finishMatch()
+            .build();
+
+    Match activeMatch = game.getLastMatch().orElseThrow();
+    List<Hand> hands =
+        activeMatch.getHands().stream()
+            .sorted(Comparator.comparing(hand -> hand.getParticipation().getParticipationNumber()))
+            .collect(Collectors.toList());
+    Hand firstHand = hands.get(0);
+    Hand secondHand = hands.get(1);
+    Hand thirdHand = hands.get(2);
+
+    assertThat(firstHand.isTurnActive(), is(false));
+    assertThat(secondHand.isTurnActive(), is(true));
+    assertThat(thirdHand.isTurnActive(), is(false));
```

Comment:
This test does not show that the turn was shifted in the ascending order, because we don't know, if in the first round the third hand or the first hand was active.

Reply:
Accepted.
But what I don't understand now: how did the_first_player_must_play_a_card_when_round_starts previously work as such, when the hands were sorted in descending order? Wouldn't then have the secondHand been in turn active instead of the first?

Reply:
bad explanation by me.
It starts with the participationnumber 0, but then continues at the end (4) and works it's way up to 0 again

Reply:
Don't get it. Why is end the number 4? There are only 3 hands i.e. participations?
If you say, it starts with partNumber 0 and then jumps to the last one (in my understanding no 2): Then in the second match the thirdHand would start and not the secondHand (like this test asserts)?
But maybe we better clarify at 3 :)